### PR TITLE
#135 Reference AssemblyExposer.MsBuild.Task as nuget dependency instead of compiling.

### PR DESCRIPTION
**Pull request by zuev93**
Code has been moved to:
https://github.com/onimp/AssemblyExposer.MSBuild.Task

Package is published here
https://www.nuget.org/packages/AssemblyExposer.MSBuild.Task/0.1.3

### CODE REVIEW

File: src/MultiplayerMod/MultiplayerMod.csproj [hunk 1]
```diff
@@ -61,4 +61,7 @@
     <ItemGroup>
         <ProjectReference Include="..\MultiplayerMod.AttributeProcessor\MultiplayerMod.AttributeProcessor.csproj" Private="false"/>
     </ItemGroup>
+    <ItemGroup>
+      <PackageReference Include="AssemblyExposer.MSBuild.Task" Version="0.1.3" Private="true"/>
```
Comment (polycone) [inline src/MultiplayerMod/MultiplayerMod.csproj:65]: Different indentation, could you please check if it's possible to configure it via `.editorconfig`? It has `indent_size = 4`, but seems like there's no effect on `.csproj` files?

File: src/MultiplayerMod/Directory.Build.targets [hunk 2]
```diff
@@ -111,7 +111,7 @@
         <ExposeAssembly SourceAssemblies="@(ExposeAssemblies)" Rules="@(ExposureRules)"/>
     </Target>
 
-    <Target Name="ExposeUnityAssemblies" DependsOnTargets="BuildAssemblyExposer" BeforeTargets="PreBuildEvent">
+    <Target Name="ExposeUnityAssemblies"  BeforeTargets="PreBuildEvent">
```
Comment (polycone) [inline src/MultiplayerMod/Directory.Build.targets:114]: Double space.

**Review by polycone — APPROVED** [review]
👍

File: src/MultiplayerMod/Directory.Build.targets (hunk 2 above)
Comment (zuev93) [reply src/MultiplayerMod/Directory.Build.targets:114]: Done.

**Pull request merged**